tutorials/users/ide/kernels-and-dependency-management.md

```diff
@@ -1 +1,85 @@
-# kernels-and-dependency-management
+# Kernels and Dependency Management
+
+## What is a Kernel?
+
+Kernels in Jupyter are self-contained, program language specific processes that run independently.  
+Having kernels configured and packaged for specific purposes gives the Jupyter environment some flexibility.
+A kernel doesn't need to be configured to serve every purpose imaginable. Several kernels can be made, with none
+of the dependencies for one kernel affecting the dependencies for another. If a kernel needs to be updated, changes can 
+be made without affecting other kernels.  If it is no longer needed, it can be deleted without 
+impacting other kernels.
+
+Below are examples of kernels available on Marble.  These will be updated or changed to fit the needs of the users.
+
+![Kernels](images/version-control/jupyter-notebook-buttons.png)
+
+## How to install packages
+
+```{warning}
+If the server is reset all additional installed packages will be uninstalled.
+```
+
+The kernels created for Marble use Python as the programming language.  As such, the packages installed are Python packages.
```

Comment:
Right now we only have Marble-specific python kernels but we will have R at least very soon.
It's fine to only include python examples in this tutorial but I'd get rid of the statement that we only use python

Reply:
Changed this sentence so it is non-specific but lists Python and R as examples.